reefer_monitoring/v1/reefer_v1.0.0-Beta-1.yaml

```diff
@@ -0,0 +1,524 @@
+        o2:
+          allOf:
+            - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/3.0.0#/components/schemas/measuredO2'
+        co2:
+          allOf:
+            - $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_DOMAIN/3.0.0#/components/schemas/measuredCo2'
+        humidity:
+          type: number
+          format: float
+          minimum: 0
+          maximum: 100
+          description: |
+            The percentage of the controlled atmosphere humidity target value
+          example: 95.6
+    controller:
+      type: object
+      x-stoplight:
+        id: 425fbc59eb757
+      properties:
+        dateTime:
+          $ref: '#/components/schemas/controllerDateTime'
+        hardwareVersion:
+          $ref: '#/components/schemas/controllerHardwareVersion'
+        maker:
+          $ref: '#/components/schemas/controllerMaker'
+        model:
+          $ref: '#/components/schemas/controllerModel'
+        operatingState:
+          $ref: '#/components/schemas/controllerOperatingState'
+        serialNumber:
+          $ref: '#/components/schemas/controllerSerialNumber'
+        softwareVersion:
+          $ref: '#/components/schemas/controllerSoftwareVersion'
+    controlledAtmosphereMode:
+      type: string
+      x-stoplight:
+        id: 78e1cd0fd35b0
+      description: The controlled atmosphere mode setting
+    equipmentReference:
+      type: string
+      x-stoplight:
+        id: 466cdf45dc3cd
+      pattern: ^\S+(\s+\S+)*$
+      maxLength: 11
+      description: |
+        The unique identifier for the equipment, which should follow the BIC ISO Container Identification Number where possible.
+        According to ISO 6346, a container identification code consists of a 4-letter prefix and a 7-digit number (composed of a 3-letter owner code, a category identifier, a serial number, and a check-digit).
+
+        If a container does not comply with ISO 6346, it is suggested to follow [Recommendation #2: Containers with non-ISO identification](https://smdg.org/documents/smdg-recommendations) from SMDG.
+      example: APZU4812090
+    activeAlarmCount:
+      type: integer
+      x-stoplight:
+        id: 907293fff9a00
+      description: The number of active alarms from the controller
+    alarmNumber:
+      type: string
+      x-stoplight:
+        id: 961e16039a0db
+      description: Harmonised Alarm Code across all Manufacturers
+    alarmDate:
+      type: string
+      format: date-time
+      description: The datetime of when the alarm first appeared
+    alarms:
+      type: object
+      x-stoplight:
+        id: ed63afe9afbf6
+      properties:
+        activeAlarmCount:
+          $ref: '#/components/schemas/activeAlarmCount'
+        activeAlarms:
+          type: array
+          items:
+            allOf:
+              - $ref: '#/components/schemas/alarmDate'
+              - $ref: '#/components/schemas/alarmNumber'
```

Comment:
Be carefull when specifying it like this. In order not to be ambiguous - I would create an object with the 2 attributes - and specify the object as the items in the array (instead of the `allOf` structure you have done now)